docs/tutorials/contract-deployment.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Deploying an EMP Contract
+sidebar_label: Expiring Multi-Party
+---
+
+This tutorial will show you how to deploy an Expiring Multi Party (EMP) contract using Etherscan or existing deployment scripts. By following these instructions, you will be able to launch an EMP on the Kovan testnet or Ethereum mainnet.
+
+This is the high-level EMP deployment process that you will need to follow.
+
+- Verify that your desired [price identifier](https://docs.umaproject.org/governance/adding-price-id#what-is-a-price-identifier) and collateral type is available on the network you wish to deploy your contract to.
+- Write your contract parameters.
+- Deploy your contract with Etherscan OR
+- Deploy your contract with a deployment script.
+
+:::danger
+
+Before deploying a mainnet contract, it is highly recommended that you have tested liquidation and dispute bots ready to deploy. Without a network of liquidation and dispute bots, a mainnet contract is vulnerable to attacks.
+
+:::
+
+## Is your desired price identifier and collateral type supported?
+Before deploying a contract, you should verify that your desired price identifier and collateral currency is already approved on the network you are trying to deploy to.
+
+View approved price identifiers and currencies here:
+- [Approved mainnet collateral currencies](https://docs.umaproject.org/oracle/mainnet-info#current-dvm-parameters)
```

Comment:
I went and looked at this link and it shows the a list of collaters & final fees but not a simple list of collateral currencies. Is the reader simply meant to read from the list under the Final fee sections?

Reply:
Yes, I will specify!